src/pages/{mdx.frontmatter__permalink}.js

```diff
@@ -20,9 +20,37 @@ const BlogPost = ({ data }) => {
   const authorBlog = AuthorSerializer.deSerialize(author);
   const { setState } = useContext(AppContext);
   const [ disappearSocial, setDisappearSocial ] = useState(false);
-
   const categoryBlog = useCategory(data.mdx.frontmatter.category);
 
+
+  const checkWindow = () => {
+    if (typeof window !== 'undefined') {
+      return window.location.pathname;
+    }
+    return;
+  }
+
+  const shareBlogPostLinks = [
+    {
+      path: `https://twitter.com/intent/tweet?url=URL&text=${checkWindow()}`, 
+      icon: <TwitterIcon className={styles.socialIcon} />,
+    },
+    {
+      path: `https://www.facebook.com/sharer/sharer.php?u=https://blog.xmartlabs.com/blog${checkWindow()}`, 
+      icon: <Facebook className={classnames(styles.socialIcon, styles.facebookSpace)} />
+    },
+    {
+      path: `https://www.linkedin.com/sharing/share-offsite/?url=https%3A%2F%2Fblog.xmartlabs.com%2Fblog%2F${checkWindow()}%2F`,
+      icon: <Linkedin className={styles.socialIcon} />
+    }
+  ];
+  
+  const shareXlProfileLinks = [
+    {path: "https://twitter.com/xmartlabs", icon: <TwitterIcon className={styles.socialIcon} /> },
+    {path: "https://es-la.facebook.com/xmartlabs/", icon: <Facebook className={classnames(styles.socialIcon, styles.facebookSpace)} />},
+    {path: "https://www.linkedin.com/company/xmartlabs/mycompany/", icon: <Linkedin className={styles.socialIcon} />}
+  ];
```

Comment:
You always use styles.socialIcon, is possible to move this into the SocialElement?

Reply:
I did something, but I don't know if it is correct.

Reply:
done?